mne/epochs.py

```diff
@@ -3672,7 +3672,7 @@ def average_movements(epochs, head_pos=None, orig_sfreq=None, picks=None,
 @verbose
 def make_fixed_length_epochs(raw, duration=1., preload=False,
                              reject_by_annotation=True, proj=True, overlap=0.,
-                             verbose=None):
+                             id=1, verbose=None):
```

Comment:
I'd rather we used a more explicit name. Thinking of `event_id` – even though this is already used elsewhere in MNE with a different meaning.

Reply:
I'm in favor of `id` to match the argument of `mne.make_fixed_length_events`.

Reply:
> to match the argument of `make_fixed_length_events`

ok, so we have precedence. In this case, let's just stick with `id`.

Reply:
yes agreed